pythonforandroid/bootstrap.py

```diff
@@ -269,14 +355,11 @@ def strip_libraries(self, arch):
             return
         strip = sh.Command(strip)
 
-        if self.ctx.python_recipe.name == 'python2':
-            filens = shprint(sh.find, join(self.dist_dir, 'private'),
-                             join(self.dist_dir, 'libs'),
-                             '-iname', '*.so', _env=env).stdout.decode('utf-8')
-        else:
-            filens = shprint(sh.find, join(self.dist_dir, '_python_bundle', '_python_bundle', 'modules'),
-                             join(self.dist_dir, 'libs'),
-                             '-iname', '*.so', _env=env).stdout.decode('utf-8')
+        filens = shprint(sh.find, join(self.dist_dir, '_python_bundle',
+                                       '_python_bundle', 'modules'),
+                         join(self.dist_dir, 'libs'),
+                         '-iname', '*.so', _env=env).stdout.decode('utf-8')
```

Comment:
here the logic is unified for both `python2` and `else` and that works? Then that's actually is a good thing and might be something we want to isolate/demonstrate in a dedicated PR